src/content/app/tools/blast/types/blastSettings.ts

```diff
@@ -33,7 +33,7 @@ const blastParameterNames = [
 ] as const;
 
 export type BlastParameterName = typeof blastParameterNames[number];
-export type SequenceType = 'dna' | 'protein';
+export type SequenceType = 'dna' | 'cdna' | 'pep';
```

Comment:
Here and elsewhere: the allowed values for the sequence type parameter in BLAST are either "dna" or "protein". See:
- [docs](https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?pageId=94148074#NCBIBLAST+HelpandDocumentation-Top)
- [open api response](ebi.ac.uk/Tools/services/rest/ncbiblast/parameterdetails/stype)

Reply:
Thats right. Tools API needs dna, cdna or pep probably to create the database path. May be @veidenberg can map this at the BE?

Reply:
Yes, 'dna' | 'cdna' | 'pep' is for selecting the target database type (in addition of the species).